fairroot/base/steer/FairTypes.h

```diff
@@ -0,0 +1,106 @@
+/********************************************************************************
+ * Copyright (C) 2014-2023 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  *
+ *                                                                              *
+ *              This software is distributed under the terms of the             *
+ *              GNU Lesser General Public Licence (LGPL) version 3,             *
+ *                  copied verbatim in the file "LICENSE"                       *
+ ********************************************************************************/
+/*
+ * FairTypes.h
+ *
+ *  Created on: May 16, 2023
+ *      Author: R.Karabowicz
+ */
+
+#ifndef FAIR_TYPES_H_
+#define FAIR_TYPES_H_
+
+#include <iostream>
+#include <limits>
+
+/**
+ * \brief Main namespace for FairRoot classes / types
+ *
+ * All new public classes and types of FairRoot will be in this namespace.
+ * Note: Nost classes are still in no namespace.
+ */
+namespace FairRoot {
```

Comment:
Didn't we agree on a lowercase name for this?
```suggestion
namespace fairroot {
```
If we didn't, I would still suggest to use lowercase. Checking the guidelines of Mozilla (upon which we base our clang-format), they seem to suggest lowercase: https://firefox-source-docs.mozilla.org/code-quality/coding-style/coding_style_cpp.html#c-namespaces

So do the Alice guidelines: https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html?showone=Namespace_Names#Namespace_Names

For me it's more a personal preference. But one objective reason could be so they are distinguishable from class names.